tests/unit/store/peek-record-test.js

```diff
@@ -41,3 +41,20 @@ test('peekRecord should return null if the record is not in the store ', functio
     assert.equal(null, store.peekRecord('person', 1), 'peekRecord returns null if the corresponding record is not in the store');
   });
 });
+
+test('peekRecord should assert if not passed both model name and id', function(assert) {
+  run(() => {
+    assert.expectAssertion(() => {
+      store.peekRecord('my-id');
+    }, /You need to pass both a model name and id/);
+  });
+});
+
+test('peekRecord should assert if passed a model class instead of model name', function(assert) {
```

Comment:
@bantic could you use `testInDebug` instead of `test` for these tests. The assert gets stripped for the production build and is causing the CI to fail when we run tests in production mode.

`import testInDebug from 'dummy/tests/helpers/test-in-debug';`